infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -15,9 +15,7 @@
 @Slf4j
 public abstract class AbstractServer {
 
-    @Getter
-    @Setter
-    volatile boolean shutdown;
+    private volatile boolean shutdown;
```

Comment:
setters and getters used only in child classes, and it was inconsistent, you could execute shutdown() method or setShutdown() both methods make the same. So actually we don't need the setter method because we have `shutdown()`. Regarding getter - it needed for CorfuMsgHandler, to check if shutdown is enable, since I extracted that check and moved it to AbstarctServer we don't need getter anymore

Reply:
would it be possible for a subclass to setShutdown but allow others to actually shutdown?

Reply:
no

Reply:
if I rename it to `isShutdown` then a getter will be `isIsShutdown` :)